src/components/icons/default-avatar.tsx

```diff
@@ -0,0 +1,7 @@
+export const DefaultAvatar = (props: React.SVGProps<SVGSVGElement>) => {
```

Comment:
This is just a user icon. Let's call it `UserIcon`.

Reply:
renamed